docs/docs/event-brokers.mdx

```diff
@@ -217,3 +227,33 @@ event_broker:
 
 With this configuration applied, Rasa will create a table called `events` on the database,
 where all events will be added.
+
+## FileEventBroker
+
+It is possible to use the `FileEventBroker` as an event broker. This implementation will log events to a file in json format.
+You can provide a path key in the `endpoints.yml` file if you wish to override the default file name: `rasa_event.log`.
+
+## Custom Event Broker
+
+If you need an event broker which is not available out of the box, you can implement your own.
+This is done by extending the base class `EventBroker`.
+
+Your custom event broker class must also implement the following base class methods:
+- `from_endpoint_config`: creates an `EventBroker` object from the endpoint configuration.
+You can find the method's signature [here](https://github.com/RasaHQ/rasa/blob/main/rasa/core/brokers/broker.py#L45).
+- `publish`: publishes a json-formatted [Rasa event](https://rasa.com/docs/rasa/reference/rasa/shared/core/events/) into an event queue.
+You can find the method's signature [here](https://github.com/RasaHQ/rasa/blob/main/rasa/core/brokers/broker.py#L63).
```

Comment:
The links are great, since it's a bit redundant to have this sentence everytime maybe just put 
```suggestion
[(see signature)](https://github.com/RasaHQ/rasa/blob/main/rasa/core/brokers/broker.py#L63).
```
or 






```suggestion
[(source - see for signature)](https://github.com/RasaHQ/rasa/blob/main/rasa/core/brokers/broker.py#L63).
```